pkg/finalize_bundle.go

```diff
@@ -104,19 +105,25 @@ func marshalJsonFile(path string, manifest interface{}) error {
 	if err != nil {
 		return err
 	}
-	return writeFile(path, manifest, mfstBytes)
+	return writeFile(path, mfstBytes)
 }
 
 func marshalYamlFile(path string, str interface{}) error {
 	mfstBytes, err := yaml.Marshal(str)
 	if err != nil {
 		return err
 	}
-	return writeFile(path, str, mfstBytes)
+	return writeFile(path, mfstBytes)
 }
 
-func writeFile(path string, str interface{}, content []byte) error {
-	err := ioutil.WriteFile(path, content, 0644)
+func writeFile(path string, content []byte) error {
+	dir := filepath.Dir(path)
+	err := os.MkdirAll(dir, 0700)
```

Comment:
the directory should already exist, if we're updating the file in place

Reply:
the `build` dir may or may not exist. With the newly added `clean` target, it will never exist.